components/badge/script.js

```diff
@@ -0,0 +1,11 @@
+const inputOne=document.querySelector("#btn-ide");
```

Comment:
You have given "#btn-ide" here, it's a typo. That's why your function was not working with the add event listener.

Reply:
i change the name of it